netlify.toml

```diff
@@ -9,3 +9,10 @@
   to = "/"
   status = 200
   force = true
+
+[[redirects]]
```

Comment:
Tidy up.

Reply:
Forgot to push recent version of this file. Fixing it.

Reply:
fixed